project/plugins.sbt

```diff
@@ -1,10 +1 @@
-// sbt-scalahost enables generation of a semantic database,
-// i.e. a storage for semantic information about Scala code.
-// See https://github.com/scalameta/scalameta/issues/605 for details.
-// The latest stable and pre-release version numbers can be found at http://scalameta.org/.
-addSbtPlugin("org.scalameta" % "sbt-scalahost" % "1.8.0")
-
-// sbt-coursier parallelizes downloads of dependencies.
-// If you haven't tried it yet, do so - the productivity boost it provides is insane.
-// It's especially useful for scala.meta, which contains more than a dozen of modules.
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC3")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
```

Comment:
Why not coursier?

Reply:
coursier is not necessary to use semanticdb.

Reply:
Shouldn't we advertise it anyway? Downloading all the dependencies without it is super annoying.

Reply:
I think we should try to keep this template as minimal as possible, it should show the absolute minimum necessary to get started.

Reply:
I feel like coursier is a distraction from that goal. The repo is so small that the download time is not a problem, this repo is not used for regular development.